client/src/app_data/pages_info/index.js

```diff
@@ -0,0 +1,32 @@
+import Home from '../../components/Home';
+import WorldWeather from '../../components/WorldWeather';
+import Feedback from '../../components/Feedback/';
+import AboutUs from '../../components/AboutUs/';
+
+export const app_name = 'Weather App';
```

Comment:
By lots of conventions it is better to name constants in upper case. 
APP_NAME, PAGES_INFO.... ROUTES maybe?

Reply:
Fixed